### DISCUSSION TITLE
I can't boot shims

### DISCUSSION THREAD

**Discussion by schoolexploitkid** [3kh0/ext-remover#1199] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
When I try to boot Shimboot it says no valid recovery image detected. I used boot shims method, but still doesn't work. My version is: 
Google Chrome	129.0.6668.80 (Official Build) (64-bit) 
Revision	07bc92bb0b0a5cf529c59d39f8167a360388904d-refs/branch-heads/6668@{#1485}
Platform	16002.44.0 (Official Build) stable-channel nissa
Firmware Version	Google_Craask.15217.607.0
Customization ID	craasula

**Comment by schoolexploitkid** [comment]
also codename is craasneta, acer chromebook 314

**Reply by qloakongithub** [reply, marked as answer]
try downgrading; pretty sure they patched booting any non-chrome images past a certain version. not sure of the specific version but i just do v101 just to be safe

**Reply by schoolexploitkid** [reply]
:v kernver 4

**Reply by qloakongithub** [reply]
yeah you prob won't be able to, pretty sure it's kenvar 3/2 and below (not completely sure because haven't been in the community in a bit but that's what i've heard). join the tn server for better help tho

**Reply by Isuckatnames308** [reply]
I have a kernver of 4 and shimboot works perfectly fine, did you remember to boot into dev mode before putting the image into it?
